supervisor/plugins/dns.py

```diff
@@ -178,8 +178,8 @@ async def update(self, version: Optional[AwesomeVersion] = None) -> None:
         try:
             await self.instance.update(version, image=self.sys_updater.image_dns)
         except DockerError as err:
-            _LOGGER.error("CoreDNS update failed")
-            raise CoreDNSUpdateError() from err
+            raise CoreDNSUpdateError(f"CoreDNS update failed", _LOGGER.error) from err
```

Comment:
```suggestion
            raise CoreDNSUpdateError("CoreDNS update failed", _LOGGER.error) from err
```

Reply:
Sorry for that mistake !! Will take care henceforth, thanks!